### PR TITLE
[READY] Blog Gebruikersoverleg September 2016

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add blog post

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
@jwcoens

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown [hunk 1]
```diff
@@ -10,18 +10,19 @@ image: images/social/gebruikersoverleg-september-2016-1200x630.png
...
+[LearningSpaces](http://www.learningspaces.nl){:target="_blank"} ondersteunt in het delen van relevante kennis. In (zelfsturende) teams, organisaties en op individueel niveau wordt het leren en kennis delen gestimuleerd. Joep belicht een aantal functionaliteiten van LearningSpaces, waaronder de Questions & Answers. Met deze functionaliteit kun je vragen naar de kennis binnen je organisatie. Wanneer je zelf vaak antwoord geeft op bepaalde vragen, kun je naast je vaste rol ook laten zien waar je competenties liggen.
+
+Team LearningSpaces werkt momenteel hard aan de nieuwe functionaliteit **Groups**. Met Groups wordt het mogelijk gemaakt kennis te groeperen, waardoor de relevantie hiervan nog groter wordt binnen je organisatie. Wil je meer weten wat je met LearningSpaces kunt? Neem dan contact op met Joep door een e-mail te sturen naar [joep@defacto.nl](mailto:joep@defacto.nl) of hem te bellen op 050 314 4832.
```
Comment (fatboypunk) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: `waardoor de relevantie hiervan nog groter wordt` isn't it more that you'll just be more focussed on that?

`Met Groups wordt het mogelijk gemaakt kennis te groeperen` would it be nice to explain that you can group things based on departments in your company? That why the follow up could be written more towards increasing the value for a person as they'll only see interesting stuff and on the otherside that it's possible to work with multiple teams while having all your company's knowledge in one platform

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown [hunk 2]
```diff
@@ -10,18 +10,19 @@ image: images/social/gebruikersoverleg-september-2016-1200x630.png
 
 **Waar normaliter de ervaringen van de gebruikers worden gedeeld, was dit keer [Defacto](/over-ons/) aan het woord. In het mooie auditorium van het Gelre Ziekenhuis hebben Gerwin, Bastiaan en Joep presentaties gehouden over het Kwaliteitspaspoort, onze applicatie 360˚Feedback en over ons product LearningSpaces.**
 
-Naast de presentaties, is een van de vaste en populaire onderdelen de zogenaamde 'Rondetafelgesprekken'. Tijdens deze gesprekken wordt dieper ingegaan op de onderwerpen en worden vragen en oplossingen gedeeld met elkaar. Het Gebruikersoverleg is altijd op locatie van een van onze klanten. Vandaag waren we te gast bij het [Gelre Ziekenhuis](https://www.gelreziekenhuizen.nl/Gelreziekenhuizen){:target="_blank"} .
+Naast de presentaties, is een van de vaste en populaire onderdelen de zogenaamde 'Rondetafelgesprekken'. Tijdens deze gesprekken wordt dieper ingegaan op de onderwerpen en worden vragen en oplossingen gedeeld met elkaar. Het Gebruikersoverleg is altijd op locatie van een van onze klanten. Vandaag waren we te gast bij het [Gelre Ziekenhuis](https://www.gelreziekenhuizen.nl/Gelreziekenhuizen){:target="_blank"}.
```
Comment (fatboypunk) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: `en worden vragen en oplossingen` two times `en`

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown [hunk 3]
```diff
@@ -10,18 +10,19 @@ image: images/social/gebruikersoverleg-september-2016-1200x630.png
...
+Feedback vragen over hoe je je werk doet kan best spannend zijn, toch kan het van grote waarde zijn voor je ontwikkeling. Bastiaan legt uit dat je met de applicatie 360˚Feedback, ontwikkeld door Defacto, gemakkelijk feedback kunt vragen aan de mensen met wie je samenwerkt, zowel intern als extern. De applicatie is ontwikkeld vanuit de aandacht voor de talentontwikkeling van de individuele medewerker. Meer weten over wat de applicatie 360˚Feedback voor jouw organisatie kan betekenen? Neem dan contact op met Bastiaan door een e-mail te sturen naar [bastiaan@defacto.nl](mailto:bastiaan@defacto.nl) of hem te bellen op 050 314 4832.
```
Comment (fatboypunk) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: A link to 360feedback.io?

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown [hunk 4]
```diff
@@ -10,18 +10,19 @@ image: images/social/gebruikersoverleg-september-2016-1200x630.png
 
 **Waar normaliter de ervaringen van de gebruikers worden gedeeld, was dit keer [Defacto](/over-ons/) aan het woord. In het mooie auditorium van het Gelre Ziekenhuis hebben Gerwin, Bastiaan en Joep presentaties gehouden over het Kwaliteitspaspoort, onze applicatie 360˚Feedback en over ons product LearningSpaces.**
```
Comment (jwcoens) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: **Delete**: Waar normaliter de ervaringen van de gebruikers worden gedeeld, was dit keer [Defacto](/over-ons/) aan het woord.

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown (hunk 3 above)
Comment (RonaldDijkstra) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: Done

File: source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown (hunk 4 above)
Comment (RonaldDijkstra) [inline source/blog/nl/2016-10-05-gebruikersoverleg-22-september-2016.html.markdown]: Done

**Pull request merged**